Scille/parsec-cloud
# [MS] Adds accept tos modal

Closes #7633 
Depends on #8471 

To test:
1. Don't forget to recompile, `python make.py i && python make.py ei`
2. Start the testbed
3. Open electron. On the device page, note the org name
4. Use the little script `python update_tos.py <OrgName>`
5. Log in to the org

Don't forget to change the locale to check if the link is updated, you can add additional locales to update_tos.py, change their name, use `en_US` or `en-us` or `en` or some kind of variations to check if the GUI manages to find the most appropriate version.

```python
# upload_tos.py. It needs requests.
import requests
import sys

response = requests.patch(
    f"http://localhost:6770/administration/organizations/{sys.argv[1]}",
    headers={"Authorization": f"Bearer s3cr3t"},
    json={"tos": {
        "en_US": "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
        "fr_BE": "https://www.youtube.com/watch?v=hEmODTcKJmE",
    }},
)

print(response.status_code, response.content)
```


- [X] Keep changes in the pull request as small as possible
- [X] Ensure the commit history is sanitized
- [X] Give a meaningful title to your PR
- [X] Describe your changes
- [X] Link any related issue in the description
